go.mod

```diff
@@ -50,6 +50,7 @@ require (
 	github.com/uber/jaeger-client-go v2.16.0+incompatible // indirect
 	github.com/uber/jaeger-lib v2.2.0+incompatible
 	github.com/uber/tchannel-go v1.10.0
+	github.com/wadey/gocovmerge v0.0.0-20160331181800-b5bfa59ec0ad // indirect
```

Comment:
You need to add an entry in the https://github.com/open-telemetry/opentelemetry-collector/blob/master/internal/tools.go to ensure we can directly manage the version of the gocovmerge.

Reply:
done.